src/db/manifest/models.rs

```diff
@@ -0,0 +1,31 @@
+use serde_derive::{Serialize, Deserialize};
+
+use crate::db::manifest::schema::manifest;
+
+#[derive(Queryable, Deserialize, Serialize, Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub struct Manifest {
+    id: i32,
+    name: String
+}
+
+impl Manifest {
+    pub fn id(&self) -> &i32 {
```

Comment:
Types that implement `Copy` (like `i32`) should probably be returned by copy instead of being returned by reference.